sopel/modules/currency.py

```diff
@@ -125,14 +128,18 @@ def exchange(bot, match):
 
     try:
         update_rates(bot)  # Try and update rates. Rate-limiting is done in update_rates()
-    except requests.exceptions.RequestException as e:
+    except requests.exceptions.RequestException as err:
         bot.reply("Something went wrong while I was getting the exchange rate.")
-        LOGGER.error("Error in GET request: {}".format(e))
+        LOGGER.error("Error in GET request: {}".format(err))
         return NOLIMIT
     except ValueError:
         bot.reply("Error: Got malformed data.")
         LOGGER.error("Invalid json on update_rates")
         return NOLIMIT
+    except APIError as err:
+        bot.reply('Sorry, something went wrong with fixer')
```

Comment:
It's not necessarily Fixer, though, is it? Could be exchangeratesapi

Reply:
Exchangeratesapi doesn't have the success field. Looking through the source it seems there's no special way for it to fail, other than standard http return codes. As such the raise_for_status() will catch it.

As discussed in IRC I'll rename the exception to better reflect that it's Fixer specific.